apps/src/blockly/constants.js

```diff
@@ -21,6 +26,13 @@ export const Themes = {
   TRITANOPIA_DARK: 'cdotritanopiadark'
 };
 
+export const Renderers = {
+  GERAS: 'cdo_renderer_geras',
+  THRASOS: 'cdo_renderer_thrasos',
+  ZELOS: 'cdo_renderer_zelos',
+  DEFAULT: 'cdo_renderer_thrasos'
```

Comment:
@breville @sanchitmalhotra126 see also ^

It's possible for Music Lab to still use Thrasos without implicitly agreeing that Music Lab is using whatever our default happens to be. Let us know if you have any concerns!